godtools/App/Features/AppLanguage/Presentation/DownloadableLanguages/DownloadableLanguagesViewModel.swift

```diff
@@ -61,9 +63,16 @@ class DownloadableLanguagesViewModel: ObservableObject {
             .store(in: &cancellables)
         
         Publishers.CombineLatest(
-            $downloadableLanguages.eraseToAnyPublisher(),
-            $activeDownloads.eraseToAnyPublisher()
+            $searchText.eraseToAnyPublisher(),
+            $downloadableLanguages.eraseToAnyPublisher()
         )
+        .flatMap({ searchText, downloadableLanguages in
+            
+            return Publishers.CombineLatest(
+                self.searchLanguageInDownloadableLanguagesUseCase.getSearchResultsPublisher(for: searchText, in: downloadableLanguages),
```

Comment:
These are difficult to spot, but we have to be careful to not capture self within a flatMap from our ViewModels.  I noticed it will create a retain cycle.  

searchLanguageInDownloadableLanguagesUseCase is simple since it's injected on init and is already in scope.

I'm wondering if you should move $activeDownloads.eraseToAnyPublisher() outside of the flatMap and include it with the searchText and downloadableLanguages?  Did you want to update display languages anytime activeDownloads is updated?